ruby/vampires.rb

```diff
@@ -0,0 +1,115 @@
+# Questions first, as the if statements below rely on the answers.
+
+puts "Good morning, Remus!  How many new employees are you processing today?"
+e = gets.to_i 
+
+while
+  e > 0
+  e -= 1
+
+  puts "What is your name?"
+  employee_name = gets.chomp 
+
+  if employee_name == "Drake Cula"
+    then employee_name = false
+  elsif employee_name == "Tu Fang"
+    then employee_name = false
+  else true
+  end
+
+
+  puts "How old are you?" 
+  age = gets.to_i
+
+  puts "What year were you born?"
+  birthyear = gets.to_i
+
+  if age == (2017 - birthyear)

```

Comment:
I would like to see you follow the age convention in the instructions.  We want to know if you the employee is lying about their age.

Lets consider checking the current year against how old they say they are and what year they say that they were born in.  Investigate the Ruby `Time` class and see if you can come up with something creative and dynamic.